scalafix-core/src/main/scala-2/scalafix/internal/config/ConfigRulePatches.scala

```diff
@@ -0,0 +1,25 @@
+package scalafix.internal.config
+
+import metaconfig.ConfDecoder
+import metaconfig.generic
+import metaconfig.generic.Surface
+import scalafix.internal.util.MetaconfigCompatMacros
+import scalafix.patch.Patch
+import scalafix.patch.Patch.internal._
+
+case class ConfigRulePatches(
+    replaceSymbols: List[ReplaceSymbol] = Nil,
+    addGlobalImports: List[AddGlobalImport] = Nil,
+    removeGlobalImports: List[RemoveGlobalImport] = Nil
+) {
+  def all: List[Patch] =
+    replaceSymbols ++ addGlobalImports ++ removeGlobalImports
+}
+
+object ConfigRulePatches {
+  implicit val surface: Surface[ConfigRulePatches] =
+    MetaconfigCompatMacros.deriveSurfaceOrig[ConfigRulePatches]
```

Comment:
something went wrong with your rebase - this file and `MetaconfigCompatMacros` shouldn't be there, and `ConfigRulePatches` should remain in `scala`